website/docs/components/accordion/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,37 @@
+## Conformance rating
+
+<!-- Update conformance rating badge with correct status and remove the others -->
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
+
+## Keyboard navigation
```

Comment:
Should we also showcase how keyboard navigation works for the `containsInteractive` variant?

Reply:
I like the idea. Just added this information to the section.